src/main/java/io/github/kawamuray/wasmtime/Trap.java

```diff
@@ -25,4 +25,11 @@ public static Trap fromExitCode(int exitCode) {
     public static Trap fromException(Throwable e) {
         return fromMessage(String.valueOf(e));
     }
+
+    public Type type() {
```

Comment:
You can add the `@Getter` annotation to the field instead.
However, I think this class is currently just a value (struct holding multiple value fields), hence it makes sense to mark this class by `@Value` (along with `@Accessors(fluent = true)`) which generates those getters automatically.